### PR TITLE
Add license field to package struct

### PR DESCRIPTION
**Pull request by ruflin**
The license field was missing in the struct but used in some packages. It defaults to basic. We need to figure out the exact values here.

### CODE REVIEW

File: util/dataset.go [hunk 1]
```diff
@@ -12,14 +12,13 @@ import (
 )
 
 type DataSet struct {
-	ID             string                   `config:"id" json:"id"`
-	Title          string                   `config:"title" json:"title" validate:"required"`
-	Release        string                   `config:"release" json:"release"`
-	Type           string                   `config:"type" json:"type" validate:"required"`
-	IngestPipeline string                   `config:"ingest_pipeline,omitempty" config:"ingest_pipeline" json:"ingest_pipeline,omitempty"`
-	Vars           []map[string]interface{} `config:"vars" json:"vars,omitempty"`
```
Comment (ruflin) [inline util/dataset.go]: This is just a cleanup I stumbled over, not related to this PR.

**Comment by ruflin** [comment]
@mtojek Rebased and also added `yaml:"license,omitempty"` for you. I assume you will need that?

**Comment by mtojek** [comment]
> @mtojek Rebased and also added `yaml:"license,omitempty"` for you. I assume you will need that?

Yes, thank you. Once regenerated, license fields will appear in manifests.

**Comment by ruflin** [comment]
@mtojek I'll leave the regeneration for a separate PR.

**Pull request merged**